### ISSUE TITLE
WIP: HTML Report

### ISSUE THREAD

**Issue by artem-zinnatullin** [gojuno/composer#11]
Sketch

![sketch](https://cloud.githubusercontent.com/assets/967132/24797751/5d4fe582-1b9b-11e7-97c3-df48fbe05b36.jpg)

Next step: collect links to good examples of coloring, fonts and overall web page styles.
Next step: discuss that with our Web Frontend Team.

**Comment by ming13** [comment]
Drone CI has some pretty pages.

<img width="1440" alt="drone" src="https://cloud.githubusercontent.com/assets/200401/24799210/b9211ed8-1ba2-11e7-97ce-2d57c2f69431.png">

I like the rectangular block, page background and search field.

![](http://www.reactiongifs.us/wp-content/uploads/2016/02/i_like_that_parks_and_rec.gif)

**Comment by artem-zinnatullin** [comment]
Crashlytics is also pretty neat, I like coloring and how easy is to focus on information

<img width="1674" alt="screen shot 2017-04-07 at 3 09 59 pm" src="https://cloud.githubusercontent.com/assets/967132/24799533/50af5994-1ba4-11e7-81cc-0aad877885dc.png">

**Comment by ming13** [comment]
Nope, it’s a mess. There is no visual focus on blocks because of the “let’s make everything white” rule.

**Comment by artem-zinnatullin** [comment]
❤️

**Comment by artem-zinnatullin** [comment]
Ok, let's stick to light main theme but dark log output. Travis also use that:

<img width="1561" alt="screen shot 2017-04-07 at 3 17 31 pm" src="https://cloud.githubusercontent.com/assets/967132/24799706/65eb7f9e-1ba5-11e7-967d-722f2af3c375.png">

**Comment by ming13** [comment]
BTW numbered lines for logcat outputs would be extremely useful for sharing.

**Comment by ming13** [comment]
The logcat should be colored as well (obviously).

![logcat-protip](https://cloud.githubusercontent.com/assets/200401/24801317/62504a8e-1bac-11e7-831c-2020e210808e.png)

**Comment by ming13** [comment]
Orkhan from our testing team added that it would be nice to have total run time displayed. Plus, showing the test package can be useful for tests grouping.

**Comment by artem-zinnatullin** [comment]
New sketch of index page we did with @navstyachka 
![2017-05-26 17 44 55](https://cloud.githubusercontent.com/assets/967132/26521559/f0380d78-42f3-11e7-83a1-2046269a3b3d.jpg)
